flox-bash/lib/commands/general.sh

```diff
@@ -1,5 +1,20 @@
 ## General commands
 
+# Set utility fallbacks
+: "${_jq:=jq}"
+: "${_nix:=nix}"
+: "${_sed:=sed}"
+: "${_column:=column}"
+: "${_grep:=grep}"
+: "${_sort:=sort}"
+: "${_gum:=gum}"
+# TODO: Don't use `nix run' to call `semver'
+if [[ -z "${_floco_uri:-}" ]]; then
+  _floco_uri=github:aakropotkin/floco
+  _floco_uri="$_floco_uri/e1231f054258f7d62652109725881767765b1efb"
+fi
+: "${_semver:=$_nix run $_floco_uri#semver --}"
```

Comment:
@limeytexan let me know what's the preferred way to call out to this. 
Should we add that repo to our lockfile/`flake.lock` and make it a `propagatedBuildInput` in the install for `flox`?

Also the block above likely needs a different fallback for `nix`. These are here to get `shellcheck` to relax, but they also raise visibility for the tooling that's in use throughout the file.